src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -955,6 +955,17 @@ private void SetSessionHandleTlsOptions(SafeWinHttpHandle sessionHandle)
                 optionData |= Interop.WinHttp.WINHTTP_FLAG_SECURE_PROTOCOL_TLS1_2;
             }
 
+            // As of Win10RS5 there's no public constant for WinHTTP + TLS 1.3
```

Comment:
We usually avoid using terms like "RS5".  Instead, we should use the marketing name "Windows 10 Version 1809".